### PR TITLE
An 810/initial setup

### PR DESCRIPTION
**Pull request by desmond-hui**
- Initial setup to run Solana Models as its own project
- Models will be built in `SOLANA` database
- Dev clone will be auto created nightly as `SOLANA_DEV`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,32 @@
...
+## Profile Set Up
+
+#### Use the following within profiles.yml 
+----
+
+```yml
+bi_analytics:
+  target: solana_dev
+  outputs:
+    eth_dev:
```
Comment (juls858) [inline README.md:10]: can we standardize on output names? maybe: dev, prd ?

File: dbt_project.yml [hunk 2]
```diff
@@ -0,0 +1,44 @@
+# Name your project! Project names should contain only lowercase characters
+# and underscores. A good package name should reflect your organization's
+# name or the intended use of these models
+name: "solana_models"
+version: "1.0.0"
+config-version: 2
+
+# This setting configures which "profile" dbt uses for this project.
+profile: "snowflake"
```
Comment (juls858) [inline dbt_project.yml:9]: this should match the suggested profile in the readme.MD: _bi_analytics_

File: models/bronze/bronze_solana__blocks.sql [hunk 3]
```diff
@@ -0,0 +1,20 @@
+{{ config (
+    materialized = 'view', 
+    tags = ['snowflake', 'solana', 'bronze_solana', 'solana_blocks']
```
Comment (juls858) [inline models/bronze/bronze_solana__blocks.sql:3]: can we start moving away from using tags?

**Review by juls858 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (desmond-hui) [reply README.md:10]: @juls858 lets standardize to `dev` and `prod`?

File: models/bronze/bronze_solana__blocks.sql (hunk 3 above)
Comment (desmond-hui) [reply models/bronze/bronze_solana__blocks.sql:3]: Agreed here

**Review by juls858 — APPROVED** [review]

**Pull request merged**